### ISSUE TITLE
feature: simulate next event

### ISSUE THREAD

**Issue by alanbork** [cronie-crond/cronie#94]
When writing new cron files it's often nice to see what will actually happen when the next event in the crontab is reached. As far as I can tell there's no "simulation" option to make this happen. 

What I propose is a new command line argument that requests that cron run any command(s) that would be due to execute at that precise moment in time, like this: cron --simulate <timestamp>. If you leave out <timestamp> it just runs the next command due to occur, and also prints the timestamp for the next even that would follow that one. Thus, if you wanted to simulate several events in a row, you would first do

# cron --simulate
next task occurs in 3 hours and 5 minutes:
[next command would be run and the result sent to stdout]

and then cron would also print 
next event would be 2 days, 5 hours, and 1 minute later, at timestamp 341414211

so that you could continue the chain as far as you wanted.

obviously this would only run the events in the current user's crontab.

it might be nice to have the option of just printing out what the command would be, in addition to the option of actually executing the command. you could make that an interactive choice, since obviously this is not a feature that would be of use in scripts.

**Comment by ferdnyc** [comment]
> it might be nice to have the option of just printing out what the command would be, in addition to the option of actually executing the command

Yeah, in fact I don't love the idea of anything initiated by a `--simulate` flag _actually executing_ anything, since to my mind `--simulate` sounds awfully close to `--dry-run` or other, similar flags that mean, "explain, _**DON'T**_ do". I would expect a `crontab --simulate` to display some information, without anything being run. 

(And, really, it _can't_ be a "proper" simulation, because the `crontab` command is running in your user session. Whereas when Cronie starts jobs from the user crontab, they're run in a very different process environment. So, that makes any such simulation inherently flawed.)

The idea of _explaining_ what would be executed and when, that part's great. I'm all for that!
